From: Colin Ian King <colin.king@canonical.com>
Date: Mon, 25 Jan 2016 18:38:06 +0000
Subject: IB/sysfs: remove unused va_list args
Patch-mainline: v4.5-rc4
Git-commit: 9f780dab7fb5618ee8bacbf6f91832374231c30c
References: bsc#966191 FATE#320230 bsc#966186 FATE#320228

_show_port_gid_attr performs a va_end on some unused va_list args.
Clean this up by removing the args completely.

Fixes: 470be516a226e8 ("IB/core: Add gid attributes to sysfs")
Signed-off-by: Colin Ian King <colin.king@canonical.com>
Reviewed-by: Ira Weiny <ira.weiny@intel.com>
Reviewed-by: Jason Gunthorpe <jgunthorpe@obsidianresearch.com>
Signed-off-by: Doug Ledford <dledford@redhat.com>
Acked-by: Benjamin Poirier <bpoirier@suse.com>
---
 drivers/infiniband/core/sysfs.c |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/infiniband/core/sysfs.c
+++ b/drivers/infiniband/core/sysfs.c
@@ -333,7 +333,6 @@ static ssize_t _show_port_gid_attr(struc
 	union ib_gid gid;
 	struct ib_gid_attr gid_attr = {};
 	ssize_t ret;
-	va_list args;
 
 	ret = ib_query_gid(p->ibdev, p->port_num, tab_attr->index, &gid,
 			   &gid_attr);
@@ -345,7 +344,6 @@ static ssize_t _show_port_gid_attr(struc
 err:
 	if (gid_attr.ndev)
 		dev_put(gid_attr.ndev);
-	va_end(args);
 	return ret;
 }
 
